.kokoro/trampoline.sh

```diff
@@ -22,5 +22,6 @@ function cleanup() {
 }
 trap cleanup EXIT
 
+chmod +x $(dirname $0)/populate-secrets.sh
```

Comment:
Instead of making `populate-secrets.sh` executable in this script, let's commit `populate-secrets.sh` to this repo in an executable state.